Packs/MicrosoftGraphFiles/pack_metadata.json

```
@@ -2,7 +2,7 @@
     "name": "Microsoft Graph Files",
     "description": "Use the Microsoft Graph Files integration to enable your app get authorized access to files in OneDrive, SharePoint, and MS Teams across your entire organization. This integration requires admin consent.",
     "support": "xsoar",
-    "currentVersion": "1.0.2",
+    "currentVersion": "1.1.0",
```

this should be a revision because there is no change in functionality.
```suggestion
    "currentVersion": "1.0.3",
```

you will need change the release notes file name to `1_0_3.md`